source/core/coverage/ut_coverage_helper.pkb

```diff
@@ -15,6 +15,10 @@ create or replace package body ut_coverage_helper is
 
   function coverage_start(a_run_comment varchar2) return integer is
   begin
+    --those are Global Temporary tables for profiler usage only
+    delete from plsql_profiler_data;
```

Comment:
`ut_coverage_helper` runs with authid definer.
The main reason for separating `ut_coverage_helper` out of `ut_coverage` was difference in authid between those two. `ut_coverage_helper` isolates users from underlying objects so you can change `ut_coverage_helper` to use 12 R2 "coverage" or something else (in theory) and don't need to change anything else than this package.